app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -1,63 +0,0 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please update the commit message for the first commit.

1. Indicate what it does in summary.
2. Do not Close a random irrelevant pull request.